_dashboards/search-telemetry.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Search telemetry
+nav_order: 30
+---
+
+
+# About Search telemetry
+
+Search Telemetry is used to analyze performance for success or failed search requests in OpenSearch Dashboards. Telemetry data is saved in the `.kibana_1` index.
+
+Because there are thousands of concurrent search request from OpenSearch Dashboards, the large traffic causes significant load in an OpenSearch cluster.
+
+To improve performance for your OpenSearch cluster, you can turn off search telemetry.
+
+## Disable search telemetry to improve cluster performance
+
+You can suppress the search usage telemetry by enabling the `data.search.usageTelemetry.enabled` setting in your `opensearch_dashboard.yml` file.
```

Comment:
The default behavior is that disabled search usage telemetry. if customer want to enable it, they can add `data.search.usageTelemetry.enabled: true`. 

"You can **enable** the search usage telemetry by enabling the `data.search.usageTelemetry.enabled` setting in your `opensearch_dashboard.yml` file."

Reply:
all looks good except for above confusion.

Reply:
thanks for the information @Flyingliuhub!